src/java.base/share/classes/jdk/internal/lang/stable/StableValueImpl.java

```diff
@@ -102,7 +102,7 @@ public void setOrThrow(T contents) {
     @ForceInline
     @Override
     public T orElseThrow() {
-        final Object t = wrappedContentAcquire();
+        final Object t = wrappedContentsAcquire();
         if (t == null) {
             throw new NoSuchElementException("No contents set");
         }
```

Comment:
```suggestion
        if (wrappedContentsAcquire() == null) {
            throw new NoSuchElementException("No contents set");
        }
```
Same as above, simplifying the code by removing local variables that are only used once

Reply:
`t` is passed to `unwrap` on the next line: <https://github.com/openjdk/jdk/blob/28bfb5c62e30516dcc5cf476ababaf7a3f1f6705/src/java.base/share/classes/jdk/internal/lang/stable/StableValueImpl.java#L104-L110>